### PR TITLE
MGMT-13700: Add learn more link for LVMS operator

### PR DESCRIPTION
**Pull request by celdrake**
https://issues.redhat.com/browse/MGMT-13700

For LVMS, starting in OpenShift 4.12 there is documentation about the operator configuration.
Added a link for the operator.

LVMS, link is shown
lvms-with-link-4-12

LVM, link not shown
lvm-without-link-4-11

### CODE REVIEW

File: src/ocm/components/clusterConfiguration/operators/LvmCheckbox.tsx
Comment (celdrake) [inline src/ocm/components/clusterConfiguration/operators/LvmCheckbox.tsx:32]: @jgyselov There will be conflicts with your PR https://github.com/openshift-assisted/assisted-ui-lib/pull/1925.

But I still see the issue in `ProgressBarAlerts` has not been addressed, so we cannot merge it yet.

**Review by ammont82 — DISMISSED** [review]

File: src/ocm/components/clusterConfiguration/operators/LvmCheckbox.tsx
Comment (jgyselov) [inline src/ocm/components/clusterConfiguration/operators/LvmCheckbox.tsx:33]: Why not use the `ExternalLink` component?

**Review by jgyselov — CHANGES_REQUESTED** [review]

File: src/ocm/components/clusterConfiguration/operators/LvmCheckbox.tsx
Comment (celdrake) [reply src/ocm/components/clusterConfiguration/operators/LvmCheckbox.tsx]: True, it's the best way. I fixed it

File: src/ocm/components/clusterConfiguration/operators/LvmCheckbox.tsx
Comment (celdrake) [reply src/ocm/components/clusterConfiguration/operators/LvmCheckbox.tsx:32]: I will fix conflicts with that PR and the one about the tooltip after they have been merged.

**Review by ammont82 — DISMISSED** [review]

**Review by jgyselov — DISMISSED** [review]

**Review by ammont82 — APPROVED** [review]

**Review by jtomasek — APPROVED** [review]

**Review by jgyselov — APPROVED** [review]

**Pull request merged**
